rpc/message.go

```diff
@@ -268,29 +339,147 @@ type CmdArgs interface {
 	cmdArgs()
 }
 
-type Event string
+type Event struct {
```

Comment:
I'm not sure what the purpose of this is

Reply:
Related atm to keeping track of subscribed events (event name + args) and handlers. to be revisited when we decide on the approach for events going forward.